types/koa-favicon/index.d.ts

```diff
@@ -23,7 +23,11 @@ declare function favicon(path: string, options?: {
     /**
      * cache-control max-age directive in ms, defaulting to 1 day.
      */
-    maxage?: number | undefined;
+    maxage?: number;
+    /**
+     * MIME type of the file at path, defaulting to image/x-icon.
+     */
+    mime?: string;
```

Comment:
The `| undefined` is still a good idea because it allows explicitly passing `undefined` to optional parameters when `exactOptionalPropertyTypes` is enabled.

<https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes>

```suggestion
    maxage?: number | undefined;
    /**
     * MIME type of the file at path, defaulting to image/x-icon.
     */
    mime?: string | undefined;
```